tests/sentry/ratelimits/utils/test_get_rate_limit_value.py

```diff
@@ -88,31 +88,3 @@ class ChildEndpoint(ParentEndpoint):
         assert get_rate_limit_value(
             "GET", RateLimitCategory.IP, rate_limit_config
         ) == get_default_rate_limits_for_group("foo", RateLimitCategory.IP)
-
-    def test_multiple_inheritance(self):
```

Comment:
this test wasn't actually testing anything beyond "does python do inheritance" so I removed it